.gitignore

```diff
@@ -1,2 +1,3 @@
 *.out
-*.o
```

Comment:
Note for next time: try not to commit changes to files apart from the `.c` code files that you create. Will merge this PR, and add both `*.o, *.out` to `.gitignore` so this won't be an issue next time.